targets/nuttx-stm32f4/nuttx/config_mpu

```diff
@@ -0,0 +1,1673 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
@akosthekiss I know your point. You talked about consistency, not dot prefix. My comment was about the dot prefix that @tadziopazur talked. My opinion is that there is no reason to keep the config files invisible by using dot-prefix.